C̝͂õ͔l̻̾l̢̃e̹͗c̙͆ṯ̈́i̹̓n̫͊g̑͜ ̙̚w͇͌e̋ͅi̱̋r̯̓ḍ̄ ͓̈́O̬̎C̯̀R͓̐ ̅͜ģ͌l̩̓y͔̓p̨͆ĥ̜s̳͑

kba opened the pull request.

zuphilip commented:
It looks that you have deleted all other pictures (for sci-work and palma) from the repo here in the commit 5ab017c9d2b8ff888ed9c7c6d9056d522a621a9c !! Moreover, the README is not in the correct place anymore.

I don't think we need the `.xcf` files here and therefore I guess we can delete them.

LetschM commented:
The pictures for sci-work are in the branch master. Konstantin made a new separate branch "ocr characters" for the new screenshots and I did changes only on this branch. So the palma/Sci-work screenshots actually should be save. 

The README for the "whole" repo is in another Branch "create-readme". I suggest we can just merge create-readme with master or move the file manually?

Yes, the xcf files can be deleted.

kba commented:
I'll lean this up tomorrow, i.e. roll back the changes to master and clean up the ocr-characters branch. Since I will need to force-push, I will notify you so you can pull the changed history. For now, I set the master branch to protected, only @stweil @zuphilip and me can push to it.

@LetschM Please make sure to pull in the changes before you add any more files and only push to ocr-characters in the future.

LetschM committed:
Folder for screenshots of special glyphs in OCR

See https://github.com/UB-Mannheim/ocr-gt-tools/wiki/Special-Characters

kba commented:
Okay, I cleaned it up and decided on a meaningful pull request title. Will check https://github.com/UB-Mannheim/ocr-gt-tools/wiki/Special-Characters for broken links and merge this into master.

zuphilip committed:
Add another example for e-with-grave

From http://digi.bib.uni-mannheim.de/fileadmin/digi/417576986/max/417576986_0012.jpg

zuphilip committed:
Add superscripted-9

from http://digi.bib.uni-mannheim.de/fileadmin/digi/445481072/max/445481072_0024.jpg

zuphilip committed:
Update superscripted-9
